staging/src/kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1/types.go

```diff
@@ -443,6 +443,8 @@ type DataSourceSource struct {
 
 // DataSourceStatus provides the most recently observed status of the DataSource
 type DataSourceStatus struct {
+	// Source is the current source of the data referenced by the DataSource
+	Source     DataSourceSource      `json:"source"`
```

Comment:
Making this field required in the JSON, breaks API backward compatibility. Can you consider marking it as `omitempty`?

Reply:
Sure, sorry I missed it.

Reply:
@akrejcir https://github.com/kubevirt/containerized-data-importer/pull/2382